src/v1/components/data-display/FontAwesomeIcons/SearchIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { FontAwesomeIcon, FontAwesomeIconProps } from "@fortawesome/react-fontawesome";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+
+type SearchIconProps = Omit<FontAwesomeIconProps, "icon">;
+
+const SearchIcon: React.FC<SearchIconProps> = (faIconProps) => (
+  <FontAwesomeIcon icon={faMagnifyingGlass} {...faIconProps} />
```

Comment:
Yeah, I did that just in case for whatever reason anyone using this component can change the icon to a different style if they wish to do so. I will change though

Reply:
Agreed to start strict: TODO for me